### PR TITLE
Small fixes in duplicatelanguage strings and models.py

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/auth/models.py [hunk 1]
```diff
@@ -613,7 +613,7 @@ class AutoGroup(models.Model):
         verbose_name=_("Regular expression for e-mail address"),
         max_length=200,
         default="^.*$",
-        help_text=_("Users with matching e-mail address will be added to this group."),
+        help_text=_("Users with a matching e-mail address will be added to this group."),
```
Comment (comradekingu) [inline weblate/auth/models.py:616]:
```suggestion
        help_text=_("Users with an e-mail address found to match will be added to this group."),
```
Possible, but maybe more complicated?

File: weblate/templates/trans/alert/duplicatelanguage.html [hunk 2]
```diff
@@ -9,11 +9,11 @@
 
     <ul>
     {% if not analysis.monolingual %}
-      <li>{% trans "You might be using monolingual translation files as bilingual. In that case please configure the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
+      <li>{% trans "You might be using a monolingual translation files as a bilingual one. If that is the case, please set the monolingual base language file origin." %} {% documentation_icon "formats" "bimono" %}</li>
```
Comment (nijel) [inline weblate/templates/trans/alert/duplicatelanguage.html:12]: Can you please clarify why origin?

Comment (comradekingu) [reply weblate/templates/trans/alert/duplicatelanguage.html:12]: It is already "set" in a sense, it just needs pointing out.
Same as "configuring it" could also mean configuring the file itself.

Comment (comradekingu) [inline weblate/templates/trans/alert/duplicatelanguage.html:12]:
```suggestion
      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file origin." %} {% documentation_icon "formats" "bimono" %}</li>
```

Comment (nijel) [reply weblate/templates/trans/alert/duplicatelanguage.html]: It's about filling in https://docs.weblate.org/en/latest/admin/projects.html#monolingual-base-language-file in the component configuration.

File: weblate/templates/trans/alert/duplicatelanguage.html [hunk 3]
```diff
@@ -9,11 +9,11 @@
 
     <ul>
     {% if not analysis.monolingual %}
-      <li>{% trans "You might be using monolingual translation files as bilingual. In that case please configure the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
+      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file origin." %} {% documentation_icon "formats" "bimono" %}</li>
```
Comment (nijel) [inline weblate/templates/trans/alert/duplicatelanguage.html:12]: I still think the origin word is not really helpful here. It's about filling in https://docs.weblate.org/en/latest/admin/projects.html#monolingual-base-language-file in the component configuration.

```suggestion
      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
```

Comment (comradekingu) [inline weblate/templates/trans/alert/duplicatelanguage.html:12]:
```suggestion
      <li>{% trans "You might be using a monolingual translation file as a bilingual one. If that is the case, please set the monolingual base language file location." %} {% documentation_icon "formats" "bimono" %}</li>
```

File: weblate/auth/models.py [hunk 4]
```diff
@@ -613,7 +613,7 @@ class AutoGroup(models.Model):
         verbose_name=_("Regular expression for e-mail address"),
         max_length=200,
         default="^.*$",
-        help_text=_("Users with matching e-mail address will be added to this group."),
+        help_text=_("Users with an e-mail address found to match will be added to this group."),
```
Comment (comradekingu) [inline weblate/auth/models.py:616]:
```suggestion
        help_text=_("Users with e-mail addresses found to match will be added to this group."),
```

File: weblate/templates/trans/alert/duplicatelanguage.html (hunk 3 above)
Comment (comradekingu) [reply weblate/templates/trans/alert/duplicatelanguage.html]: Went with location. Anything to help the user understand the file is already there in need of pointing out.

File: weblate/auth/models.py [hunk 5]
```diff
@@ -613,7 +613,7 @@ class AutoGroup(models.Model):
         verbose_name=_("Regular expression for e-mail address"),
         max_length=200,
         default="^.*$",
-        help_text=_("Users with matching e-mail address will be added to this group."),
+        help_text=_("Users with e-mail addresses found to match will be added to this group."),
```
Comment (nijel) [inline weblate/auth/models.py:616]:
```suggestion
        help_text=_(
            "Users with e-mail addresses found to match will be added to this group."
        ),
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**